app/src/routes/transfer/(components)/transfer-form.svelte

```diff
@@ -378,23 +328,29 @@ const transfer = async () => {
     }
   } else if ($fromChain.rpc_type === "evm") {
     const connectorClient = await getConnectorClient(config)
-    const selectedChain = getChainById(Number($fromChainId))
+    const selectedChain = evmChainFromChainId($fromChainId)
+
+    const unionClient = createUnionClient({
+      account: connectorClient.account,
+      chainId: $fromChain.chain_id as EvmChainId,
+      transport: custom(window.ethereum) as unknown as HttpTransport
```

Comment:
should we do a safe check here rather than a cast?

Reply:
i'll do a nice cleanup of the form/page with xstate + proper neverthrow usage in the upcoming pr